core/src/main/java/io/grpc/internal/RetriableStream.java

```diff
@@ -777,4 +785,58 @@ public long addAndGet(long newBytesUsed) {
       return bufferUsed.addAndGet(newBytesUsed);
     }
   }
+
+  @Immutable
+  static final class RetryPolicy {
+    private final int maxAttempts;
+    private final double initialBackoffInSeconds;
+    private final double maxBackoffInSeconds;
+    private final double backoffMultiplier;
```

Comment:
backoffMultiplier should be an int

Reply:
That surprises me. Why I can not set backoffMultiplier = 1.5 if I consider 2 is too large?

Reply:
The schema from the design (and quoted in this PR in the comment block in `ManagedChannelImpl#getRetryPolicies`) says: `"backoffMultiplier": number`. I'm not aware of any strong constraint that the design couldn't be changed to allow floating point numbers here, but neither did a strong (or any) need for this come up during the design phase, so it's currently specified as an integer.

Reply:
Nevermind, I was incorrect - I hadn't realized json numbers can be either float or int. We're using the same json type for maxAttempts (which must be an integer) as for backoffMultiplier, but in the proto representation backoffMultiplier can indeed be a floating point number.